metadata-ingestion/scripts/docgen.py

```diff
@@ -993,18 +1029,15 @@ def generate(
                     )
 
         f.write("""
+        
+### SQL Parser Lineage Extraction
 
-### Types of Lineage Connections
+If you’re using a different database system for which we don’t support column-level lineage out of the box, but you do have a database query log available, 
+we have a SQL queries connector that generates column-level lineage and detailed table usage statistics from the query log.
 
-Types of lineage connections supported in DataHub and the example codes are as follows.
+If these does not suit your needs, you can use the new `DataHubGraph.parse_sql_lineage()` method in our SDK. (See the source code [here](https://github.com/datahub-project/datahub/blob/master/metadata-ingestion/src/datahub/ingestion/graph/client.py#L1132))
```

Comment:
this line number is going to go out of date

maybe link to this instead https://datahubproject.io/docs/python-sdk/clients/